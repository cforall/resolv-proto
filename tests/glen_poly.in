// based on example on p.89 of Glen's thesis
//
// Expected behaviour:
// f-f3( 1 1 )
// f-f2( 2 2 )
// f-f3( 0 0 => 1 )
// f-f2( 1 => 2 2 )
// f-f1( 1 #p )
f-f1 T U
f-f2 T T
f-f3 T 1
%%
f( 1 1 )
f( 2 2 )
f( 0 0 )
f( 1 2 )
f( 1 #p )
