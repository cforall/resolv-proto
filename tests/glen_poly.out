
[Void / (0,1,1,0,0)]{T<0> => 1} f-f3{T<0>}( 1 1 )

[Void / (0,2,1,0,0)]{T<0> => 2} f-f2{T<0>}( 2 2 )

[Void / (0,1,1,0,1)]{T<0> => 0} f-f3{T<0>}( 0 0 => 1 )

[Void / (0,2,1,0,1)]{T<0> => 2} f-f2{T<0>}( 1 => 2 2 )

[Void / (0,2,2,0,0)]{T<0> => 1, U<1> => #p} f-f1{T<0>, U<1>}( 1 #p )
